e2e_example/tool/build.dart

```diff
@@ -13,27 +13,18 @@ import 'package:shelf/shelf_io.dart' as shelf_io;
 
 Future main() async {
   var graph = new PackageGraph.forThisPackage();
-  var buildActions = <BuildAction>[
-    new BuildAction(new TestBootstrapBuilder(), graph.root.name,
+  var builders = <BuilderApplication>[
```

Comment:
Technically we shouldn't need the explicit type here I don't think?

Reply:
Done